tracee-ebpf/tracee/events_derived.go

```diff
@@ -0,0 +1,92 @@
+package tracee
+
+import (
+	"fmt"
+
+	"github.com/aquasecurity/tracee/pkg/external"
+)
+
+type deriveFn func(EventDefinition) error
+
+func (t *Tracee) deriveEvent(event external.Event) []external.Event {
+	var deriveFns map[int32]deriveFn
+	var derivatives []external.Event
+
+	eventID := int32(event.EventID)
+	switch eventID {
+	case CgroupMkdirEventID:
+		containerCreateFn := func(def EventDefinition) error {
+			cgroupId, err := getEventArgUint64Val(&event, "cgroup_id")
+			if err != nil {
+				return err
+			}
+
+			if info := t.containers.GetCgroupInfo(cgroupId); info.ContainerId != "" {
+				de := event
+				de.EventID = int(ContainerCreateEventID)
+				de.EventName = "container_create"
+				de.ReturnValue = 0
+				de.StackAddresses = make([]uint64, 1)
+				de.Args = []external.Argument{
+					{ArgMeta: def.Params[0], Value: info.Runtime},
+					{ArgMeta: def.Params[1], Value: info.ContainerId},
+					{ArgMeta: def.Params[2], Value: info.Ctime},
```

Comment:
i think it might be better that ctime will be of a numeric type (int64 probably), so we'll be able to use it later without convertions. here `info.Ctime` is `time.Time`.